tests/pytests/unit/utils/test_minions.py

```diff
@@ -8,15 +8,16 @@ def test_connected_ids():
     test ckminion connected_ids when
     local_port_tcp returns 127.0.0.1
     """
-    opts = {"publish_port": 4505}
+    opts = {"publish_port": 4505, "minion_data_cache": True}
     minion = "minion"
-    ip = salt.utils.network.ip_addrs()
-    mdata = {"grains": {"ipv4": ip, "ipv6": []}}
-    ckminions = salt.utils.minions.CkMinions({"minion_data_cache": True})
+    ips = {"192.168.1.1", "10.10.10.10"}
```

Comment:
**suggestion** use 203.0.113.0/24 or other TEST-NET addresses (https://en.m.wikipedia.org/wiki/Reserved_IP_addresses) to explicitly communicate these are not real IP addresses

Reply:
I didn't know about that there were reserved IP addresses that can be used for this purpose, that's useful. I'll change the IPs. Thanks!